zerver/views/messages.py

```diff
@@ -1642,6 +1642,7 @@ def json_fetch_raw_message(request: HttpRequest, user_profile: UserProfile,
     (message, user_message) = access_message(user_profile, message_id)
     return json_success({"raw_content": message.content})
 
+@has_request_variables
 def json_fetch_raw_list_messages(request: HttpRequest, user_profile: UserProfile,
                                 messages: List[int]=REQ(validator=check_list(check_int))
```

Comment:
There is an indentation issue here and on the line after this. You can use the lint tool with `./tools/lint` to fix and/or find issues like this in your development environment.

You can go through Zulip's [Lint tools documentation](https://zulip.readthedocs.io/en/latest/contributing/code-style.html#lint-tools) for more information.